contracts/masset/RewardManager.sol

```diff
@@ -335,7 +369,7 @@ contract RewardManager is IRewardManager, Ownable {
         require(msg.sender == address(masset), "only masset may call");
 
         uint256 reward = this.getRewardForDeposit(_bassetAddress, _sum, _bridgeMode);
-        if(reward >= MIN_REWARD) {
+        if(reward >= DUST_LIMIT) {
```

Comment:
why?

Reply:
the function called `sendRewardForDeposit` returns rewards amount even if it is dust and was not actually sent.  
this function is called from the Mynt `payReward` function [here](https://github.com/DistributedCollective/babelfish-sc/blob/28dbe23457f7ebe76842e9cbc62218166815a3af/contracts/masset/Masset.sol#L130) which emits event `onRewardPaid` even though it was not actually paid.   
and returns reward amount in any case.  
is it by design?

Reply:
This if is an optimization added later to prevent 0 sum transactions. If the reward amount is less than $1/1000000 we don't actually send the transaction. But the rest of the code treats it normally. We want the numbers to add up in the accounting. So yes, I expect it is by design.